docs/reference/ml/anomaly-detection/apis/get-category.asciidoc

```diff
@@ -85,6 +95,18 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=job-id-anomaly-detection]
 The value is increased by 10% to enable matching for similar fields that have
 not been analyzed.
 
+// This doesn't use the shared description because there are
+// categorization-specific aspects to its use in this context
+`partition_field_name`::
+(string) The field used to segment the categorization, if per-partition
+categorization was enabled. This field is not present when per-partition
```

Comment:
To avoid confusion with the "field" in the previous sentence, I'd recommend omitting the second sentence or changing it like this:
```suggestion
categorization was enabled. It is not present when per-partition
```